source/reference/minio-mc-admin/mc-admin-replicate.rst

```diff
@@ -338,6 +338,29 @@ Syntax
                                minio1         \
                                --user janedoe
 
+         The output of any of hte above examples resembles the following:
+
+         .. code-block:: shell
+
+            Bucket replication status:
+            ●  30/30 Buckets in sync
```

Comment:
OK so _fun fact_ these dots are a unicode character that, for reasons I cannot surmise, do not play well *at all* with the pandoc latex conversion pipeline. 

If you can replace these with simple hyphens it would be enough, since this is an output example.

Reply:
Replaced